docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -65,6 +66,56 @@ Applications that target versions of .NET Framework starting with 4.7.1 can disa
 </runtime>
 ```
 
+## What's new in accessibility in .NET Framework 4.8.1
+.NET Framework 4.8.1 includes new accessibility features in the following areas:
+
+- [Windows Forms](#winforms481)
+
+- [Windows Presentation Foundation (WPF)](#wpf481)
+
+<a name="winforms481"></a>
+
+### Windows Forms
+
+**Added and improved UIA representations**
+
+Prior to .NET Framework 4.8.1, Windows Forms was missing support for a range of UIA patterns to support assistive technology, for example, to interact with applications. This deficiency could cause Narrator or other screen readers to report incomplete or incorrect information. It could also affect important functionality like moving a cursor through the text in a <xref:System.Windows.Forms.TextBox> control. With .NET Framework 4.8.1, all of the required patterns for the common controls have been implemented. This new functionality gives users of assistive technology a much richer application interaction experience.
+
+- Added support for the UIA expand/collapse pattern to the <xref:System.Windows.Forms.DateTimePicker> control.
+- Added UIA support to the <xref:System.Windows.Forms.MonthCalendar> control. Now assistive technology tools such as Narrator can navigate through the individual dates in the control.
+- Implemented text pattern support for all text-based controls, including the <xref:System.Windows.Forms.TextBox>, <xref:System.Windows.Forms.MaskedTextBox>, <xref:System.Windows.Forms.PropertyGrid> edit control, <xref:System.Windows.Forms.DataGridViewTextBoxCell>, <xref:System.Windows.Forms.ToolStripTextBox>, and <xref:System.Windows.Forms.DomainUpDown> controls.
+- <xref:System.Windows.Forms.ToolTip> now follows [WCAG2.1 guidelines](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) to be persistent, dismissable, and hoverable on Windows 11.
+
+**Various bug fixes for existing accessibility features**
+
+- Narrator can now focus on an empty <xref:System.Windows.Forms.DataGridView> control.
+- Addressed an issue that caused screen readers to count hidden columns when announcing the column count in a <xref:System.Windows.Forms.DataGridView> control.
+- Addressed an issue that caused the <xref:System.Windows.Forms.DataGridView> to ignore the font settings set in the `DataGridviewCellStyle` if the underlying form had a `Font` property that differed from `DefaultFont`.
+- Updated the `AccessibleName` property of the <xref:System.Windows.Forms.DataGridView> control's internal scroll bars to remove the text "ScrollBar".
+- Fixed the color of a `DataGridViewLinkCell` when the cell is selected.
+- Fixed an issue with custom <xref:System.Windows.Forms.DataGridView> controls where there was no `ControlType` or `LocalizedControlType` provided publicly for custom <xref:System.Windows.Forms.DataGridViewCell> elements.
```

Comment:
```suggestion
- Fixed an issue with custom <xref:System.Windows.Forms.DataGridView> controls where there was no `ControlType` or `LocalizedControlType` provided for custom <xref:System.Windows.Forms.DataGridViewCell> elements.
```
Remove `publicly` - this name is not providedr as a public property, it is just set to a hardcoded value.